Jenkinsfile

```diff
@@ -8,6 +8,14 @@ devToolsProject.run(
     sh 'pipenv sync --dev'
   },
   build: { data ->
+    Map creds = encryptedFile.readJson(
+      path: 'credentials.json.enc',
+      credentialsId: 'jenkins-node-scanner-password',
+    )
+    String jinjaCommand = "jinja2 -D host=${creds['host']} -D port=${creds['port']}"
+    sh "pipenv run ${jinjaCommand} -o papertrail_config.yml papertrail_config.yml.j2"
```

Comment:
For future reference, here's a nifty Groovy trick you could use in the future:

```groovy
['papertrail_config.yml', 'supervisord.conf'].each { i ->
  sh "pipenv run ${jinjaCommand} -o ${i} ${i}.j2"
}
```

That's probably overkill when we only have 2 files here, but if we had a bunch more then you'd definitely want to use an `each` loop to save copy/pasting. Anyways, you can take or leave that suggestion as far as this commit goes.